script.sh

```diff
@@ -358,13 +353,8 @@ else
 	echo Response not recognized.
 fi
 echo "Telnet is complete."
-
-clear
-if [ $vpnYN == no ]
-then
-	apt-get purge openvpn -y -qq
```

Comment:
OpenVPN still needs to be purged, you will need to do this somewhere else instead.